src/main/resources/config.sample.yml

```diff
@@ -45,30 +48,45 @@ platforms :
       url: http://example.com/
 
     - type: pybossa
+
       # the key to access the pybossa api
       apiKey: 8ec92fa1-1bd1-42ad-8524-3d2bab4588b1
+
       # the url to the pybossa api
       url: http://localhost:5000/api
+
+      # name which is used to display
       name: Pybossa
+
       # the id of the pybossa project to use for all experiments
       projectId: 1
+
       # if calibrations are allowed on this platform
       calibrationsAllowed: true
 
 # basic configurations for the mail-module
-mail :
+mail:
```

Comment:
Sorry I meant newlines. 
And no you didnt do it infront of _ALL_ other documented options, see line 36 in the file there is still " platforms :" thats why I say its still _NOT_ consistent.